cognite/seismic/protos/persisted_trace.proto

```diff
@@ -2,12 +2,14 @@ syntax = "proto3";
 package com.cognite.seismic;
 
 /*
-   Version:
-   0 - no compression of traces
-   1 - fpzip compression of traces
-   2 - waveform compression
-
-   The trace header is gzip compressed in all versions
+Version:
+    For trace data:
+        1 - no compression of traces
+        2 - zstd compression of traces
+        3 - waveform compression
```

Comment:
Maybe the waveform compression can just be forgotten now? I can't see any remains of it.